openshift/Jenkinsfile

```diff
@@ -46,8 +46,8 @@ pipeline {
                             echo "Configure template, you can use add to project if you want to install the app in an oteher env eg debug dev etc"
                             oc project sc-avo2
                             oc -n sc-avo2 apply -f openshift/avo2-client-tmpl-dc.yaml
-                            oc process -l app=avo2-client -p ENV=dev REACT_APP_PROXY_URL=http://avo2-proxy-dev-sc-avo2.apps.do-prd-okp-m0.do.viaa.be -f openshift/avo2-client-tmpl-dc.yaml | oc apply -f -
-                            oc process -l app=avo2-client -p ENV=qas REACT_APP_PROXY_URL=http://avo2-proxy-qas-sc-avo2.apps.do-prd-okp-m0.do.viaa.be -f openshift/avo2-client-tmpl-dc.yaml | oc apply -f -
+                            oc process -l app=avo2-client -p ENV=dev PROXY_URL=http://avo2-proxy-dev-sc-avo2.apps.do-prd-okp-m0.do.viaa.be -f openshift/avo2-client-tmpl-dc.yaml | oc apply -f -
```

Comment:
@violetina Volgens mij mogen deze hier weg en mogen deze variabelen toegevoegd worden in OpenShift om at runtime geïnjecteerd te worden. (Note: Ze zijn ook hernoemd naar `PROXY_URL` ipv. `REACT_APP_PROXY_URL`).